ci/post-msg.py

```diff
@@ -14,6 +15,28 @@
     print("ERROR!  Incorrect number of arguments")
     sys.exit(1)
 
+def apply_stats(file, name):
+    global comment_body
+    global ACTION
+    if path.exists(file):
+        with open(file) as f:
+            results = json.load(f)
+        if (results['pass_performance_check']):
+            comment_body += " :heavy_check_mark: {} performance check passed\n".format(name)
+        else:
+            comment_body += " :x: PR drops {} performance below minimum requirement\n".format(name)
+            ACTION = 'REQUEST_CHANGES'
+
+def push_results(file):
```

Comment:
this doesn't actually push_results to github it just adds them to the comment so maybe add_results or process_results?